dbt-cta/impactive/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -0,0 +1,1237 @@
+version: 2
+
+models:
+
+  - name: person_api_lookups_export_base
+    description: ''
+    columns:
+      - name: person_lookup_id
+        description: ''
+      - name: created_at
+        description: ''
+      - name: exported_at
+        description: ''
+      - name: api_match_id
+        description: ''
+      - name: api_username
+        description: ''
+      - name: person_lookup_type
+        description: ''
+      - name: updated_at
+        description: ''
+      - name: api_key
+        description: ''
+      - name: api_type
+        description: ''
+      - name: success
+        description: ''
+      - name: van_database_mode
+        description: ''
+      - name: id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: campaign_id
+        description: ''
+      - name: _airbyte_ab_id
```

Comment:
I'm a bit conflicted with tests on `_airbyte_ab_id` especially if there are other columns that can be used for row uniqueness. I think the name of this column in Destinations V2 is more descriptive (`_airbyte_raw_id`) since its just the id for the record in the raw table. So the test would catch if we somehow parse the same raw record multiple times but in a full refresh model it shouldn't happen? what do you think?

Reply:
Hmmm, I'm afraid I don't know the differences between `_airbyte_ab_id` and `_airbyte_raw_id`. I've been striving to use a native primary key when possible in addition to this `_airbyte_ab_id` column and definitely think if there's a better column to use that we should! Like you said in your next comment, pershaps the `_airbyte_{table_name}_hash_id` column might be better?

Reply:
they're the same column! Airbyte just decided to rename the `airbyte_ab_id` in the newest version of destinations. But yes I think the hash_id is a better column for uniqueness check!